src/getCoinCombination.test.js

```diff
@@ -7,4 +7,32 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test('should return correct combination of coins for 0 cents', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  test('should return correct combination of coins for 5 cents', () => {
```

Comment:
The solution is good, but you need to practice a bit with the test names. It should sound like a test case - we need to understand, what exactly we are testing:
"should return 1 nickel when '5' is entered"

Reply:
Done